data/json/items/gun/ups.json

```diff
@@ -190,5 +192,47 @@
     ],
     "ammo_effects": [ "LASER", "INCENDIARY" ],
     "flags": [ "NEVER_JAMS", "NO_UNLOAD", "NON_FOULING", "NEEDS_NO_LUBE" ]
+  },
+  {
+    "id": "gatling_mech_laser_single",
+    "copy-from": "gatling_mech_laser",
+    "looks_like": "m134",
+    "type": "GUN",
+    "name": { "str": "salvaged CMES laser cannon" },
+    "description": "This gatling-laser has been stripped from its CMES mech-suit and is far too heavy to use on its own.  You will need to deploy it or mount it to a vehicle, but it could be devastating if you have deep battery reserves.",
+    "valid_mod_locations": [
```

Comment:
Does this need valid mod locations?  Do mods work when attached to vehicles?

Reply:
I *believe* they do. It's up in the air whether they should be moddable given their repurposed nature. Should I remove those entries?

Reply:
Perhaps the availability could be checked more carefully. Fastening a carry strap to it should still be trivial and the laser components need to be serviced in some fashion even on the mech, but I doubt they provide standard sight rails or a typical stock attachment without those explicitly being crafted by the player for example. Your obtainment method currently implies just taking them from the mech as-is without any additional effort or resource expenditure to render them usable for a human operator.